packages/components/organisms/f-cookie-banner/src/components/_tests/CookieBanner.test.js

```diff
@@ -0,0 +1,301 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import CookieBanner from '../CookieBanner.vue';
+import Cookie from 'cookie-universal';
+import { VueI18n } from '@justeat/f-globalisation';
+
+const localVue = createLocalVue();
+localVue.prototype.$cookies = Cookie();
+localVue.use(VueI18n);
+const i18n = {
+    locale: 'en-GB'
+};
+
+describe('CookieBanner', () => {
+    describe('components', () => {
+        it('should be defined', () => {
+            // Arrange
+            const propsData = {};
+
+            // Act
+            const wrapper = shallowMount(CookieBanner, {
+                localVue,
+                i18n,
+                propsData
+            });
+
+            // Assert
+            expect(wrapper.exists()).toBe(true);
+        });
+    });
+
+    describe('methods', () => {
+        describe('isNotExcluded()', () => {
```

Comment:
We normally don't put the `()` in the name of the tests here. Could we please remove them?

Reply:
Done